src/DependencyInjection/Configuration.php

```diff
@@ -30,6 +30,10 @@ public function getConfigTreeBuilder()
                 ->cannotBeEmpty()
                 ->defaultValue('%kernel.project_dir%/build')
             ->end()
+            ->booleanNode('shared_html_crawlers')
```

Comment:
Yes, I think this feature is reserved to advanced users that knows what they're doing.

The drawback I can identify is the following:
>As a Stenope user, if I create my own Processor and I use it to modify the html "content" property, the shared HTML Crawler feature will eventually override my changes and it would be hard for me to understand what's happening.

That's why I see this purely optimisation related feature as an opt-in for advanced users (like us) that will then know to use the `HtmlCrawlerManagerInterface` service in any custom processor that access HTML properties.

Reply:
Fair enough 👍🏻